en/core-utility-libraries/email.rst

```diff
@@ -94,6 +94,11 @@ you can also just load an array of configs::
 
     //or in constructor::
     $Email = new CakeEmail(array('from' => 'me@example.org', 'transport' => 'MyCustom'));
+    
+.. note::
+    
+    Use $Email->config() or the constructor to set the log level to log email headers and message.
```

Comment:
Please use bacticks, like so:

Use `$Email->config()` or ...

Makes it more readable.